packages/manager/src/queries/profile/preferences.ts

```diff
@@ -1,39 +1,41 @@
 import { updateUserPreferences } from '@linode/api-v4';
-import {
-  QueryClient,
-  useMutation,
-  useQuery,
-  useQueryClient,
-} from '@tanstack/react-query';
-
-import { ManagerPreferences } from 'src/types/ManagerPreferences';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
 
 import { queryPresets } from '../base';
 import { profileQueries } from './profile';
 
 import type { APIError } from '@linode/api-v4';
+import type { QueryClient } from '@tanstack/react-query';
+import type { ManagerPreferences } from 'src/types/ManagerPreferences';
 
-export const usePreferences = (enabled = true) =>
-  useQuery<ManagerPreferences, APIError[]>({
+export const usePreferences = <TData = ManagerPreferences>(
```

Comment:
Optional, just thinking of possible code documentation since this link was called out in the PR description.

```suggestion
// Reference for this pattern: https://tkdodo.eu/blog/react-query-data-transformations#3-using-the-select-option
export const usePreferences = <TData = ManagerPreferences>(
```